### ISSUE TITLE
3DS Sounds

### ISSUE THREAD

**Issue by Svilojevo** [XProger/OpenLara#316]
I got OpenLara playing on my New 3DS XL. LOVE IT!!!! What an amazing experience for TR fans. Thank you so much for all your work on this. Just WOW!!!!  Classic Lara in stereoscopic 3D. It plays smooth without any frame drops, and I can use the analog stick. 

Couple questions:

1) Is there a way to hide or turn off all the scrolling text on the bottom screen? A little distracting, and takes away from the experience.

2) I'm not getting any sounds. Is the sound not implemented or do I need to convert the sounds to be playable on the 3DS. I'm using a PSX rip.

<img width="570" alt="Screen Shot 2021-01-31 at 4 04 12 PM" src="https://user-images.githubusercontent.com/66494480/106402365-01dbbe80-63de-11eb-9316-ceba8facf634.png">

**Comment by XProger** [comment]
Thanks.
1. Just tap on the bottom screen
2. You can convert all PSX audiotracks to ogg and put it into audio folder or just unpack [this](http://xproger.info/projects/OpenLara/files/TR1_audio.7z) archive.

**Comment by Svilojevo** [comment]
This is perfect. Thank you so much. Now the text is gone. I used your archives, and everything loads great, but still no audio for some reason. The volume works on my 3DS until I start the game, then nothing. I placed the "audio" folder from the link in the OpenLara folder.

<img width="527" alt="Screen Shot 2021-01-31 at 4 39 30 PM" src="https://user-images.githubusercontent.com/66494480/106403330-e8894100-63e2-11eb-8e75-53ff81d45291.png">

**Comment by XProger** [comment]
No any sounds even while shooting? Check in-game audio settings.

**Comment by Svilojevo** [comment]
Not even while shooting.

![JPEG-image-5534FFA905DF-1](https://user-images.githubusercontent.com/66494480/106403480-ce9c2e00-63e3-11eb-806e-bfa3ae4e3fd9.jpg)

**Comment by Svilojevo** [comment]
Thanks to XProger  I figured out that it was my Homebrew issue. So on to figuring out my Homebrew sound.

**Comment by davesade** [comment]
Sorry to bother you with this. First of all - excellent work on this port. It runs smoothly on my ambassador 3DS and I'm happy to relive the experience in glorious stereoscopic 3D.

I couldn't managed to make music to play. I extracted the package you suggested and on my SD card it looks merely identically as above:

> 
> <img alt="Screen Shot 2021-01-31 at 4 39 30 PM" width="527" src="https://user-images.githubusercontent.com/66494480/106403330-e8894100-63e2-11eb-8e75-53ff81d45291.png">

I can hear normal audio sounds during the gameplay. It's just music doesn't load apparently. There is no error at all. Just for a heck of it, I tried to rename the folder audio to AUDIO, but no difference. Any advice appreciated.
